src/v2/view-builder/views/TerminalView.js

```diff
@@ -1,14 +1,32 @@
-import { createCallout } from 'okta';
+import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals/BaseFooter';
+import {getBackToSignInLink} from '../utils/LinksUtil';
 
 const Body = BaseForm.extend({
   noButtonBar: true,
+  isTerminalForExpiredLink: false,
```

Comment:
probably better to have a helper function (in separated file) which take `messagesObjs` and figure out whether it's a email link expired therefore can be reused even when creating the go back link.

Reply:
that was a trade of between calling function (computing every time) versus compute once and keep it in variable.